src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Templates/Microsoft.DotNet.UpgradeAssistant.Steps.Templates.csproj

```diff
@@ -9,6 +9,10 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="Templates\VisualBasicWebAppTemplates\Program.vb" />
```

Comment:
You can just add `<None Remove="Templates\**\*.vb" />` around line 7 or 8 instead.